.gitignore

```diff
@@ -36,6 +36,7 @@ node_modules/
 npm-debug.log
 yarn-error.log
 package-lock.json
+yarn.lock
```

Comment:
WHY?

Reply:
> we don't need ios/ folder or yarn.lock file changes for the PR please take this out

@adnanjt

Reply:
Why we are removing the `yarn.lock` file?

Reply:
@iocedano We don't need it to update the repo this can be updated on each environment and cause fewer problems. Its updated with yarn

Reply:
Con esto no me refiero a borrar el archivo completamente 

solo no subirlo